### ISSUE TITLE
CMake build task does not show progress

### ISSUE THREAD

**Issue by diggit** [microsoft/vscode-cmake-tools#2757]
### Brief Issue Summary

Executing `CMake: Build` command in VSC, results in progress being displayed on bottom VSC bar showing progress (and output in OUTPUT tab).
Executing Task with cmake build does not show progress (and output in terminal).

Is there a way, how to achieve same behavior from task?

Task
```json
{
	"label": "build",
	"detail": "CMake template build task",
	"type": "cmake",
	"command": "build",
	"targets": [
		"all"
	],
	"group": {
		"kind": "build",
		"isDefault": true
	},
	"problemMatcher": [],
	"options": {
		"cwd": "${workspaceFolder}/build"
	},
	"presentation": {
		"reveal": "silent",
		"panel":"dedicated"
	},
}
```

**Comment by elahehrashedi** [comment]
by removing this setting in your build task `"reveal": "silent"` you should be able to see the Build progress in the terminal.
for example:
![image](https://user-images.githubusercontent.com/16143955/193682683-b1a49dad-6bed-4c23-b964-9d3513607195.png)

**Comment by diggit** [comment]
Yeah, but this way, I have to always manually switch to something else manually.
